src/zcash/zecEngine.js

```diff
@@ -318,6 +317,14 @@ export class ZcashEngine extends CurrencyEngine<ZcashPlugin> {
     // **********************************
     // Create the unsigned EdgeTransaction
 
+    const spendTargets = edgeSpendInfo.spendTargets.map(si => ({
+      uniqueIdentifier: si.uniqueIdentifier,
+      memo: si.memo,
+      nativeAmount: si.nativeAmount ?? '0',
+      currencyCode,
+      publicAddress
+    }))
+
```

Comment:
What is happening here? This seems like it needs either to be in another commit so we can document in the message the change, or add a comment to explain the need for this change.

Reply:
Damn. This was an unrelated change that also snuck in. I'll put this in a new PR. Too many simultaneous changes at the same time.

Reply:
This needs to be the same commit or you'll get Flow errors. the SpendInfo.spendTargets is a different shape than EdgeTransaction.spendTargets. This line does the conversion. When super.makeSpend returned the edgeSpendInfo as type `Object` it obscured the bug.